modules/axruntime/Cargo.toml

```diff
@@ -35,7 +36,9 @@ axfs = { workspace = true, optional = true }
 axnet = { workspace = true, optional = true }
 axdisplay = { workspace = true, optional = true }
 axtask = { workspace = true, optional = true }
-axplat = "0.1"
+# wait for the PR merged and released on crates.io, modify this to axplat =  "0.2"
```

Comment:
You can add `[patch.crates-io]` section in the workspace `Cargo.toml` to avoid modifications in other `Cargo.toml`.